BREAKING.md

```diff
@@ -32,6 +33,10 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 - [Remove ISummaryBaseConfiguration.summarizerClientElection](#Remove-ISummaryBaseConfigurationsummarizerClientElection)
 - [Remove Deprecated IFluidObject Interface](#Remove-Deprecated-IFluidObject-Interface)
 
+### Sub Directory creation with same name fail in Shared Directory DDS
+If user tries to create a sub directory with name `Foo` and `Foo` sub directory already exists, then instead of submitting an OP
+unneccessarily, the `createSubDirectory` method will throw an error.
```

Comment:
what kind of error? Is it recoverable? Does it close the container?

Reply:
So, it is a user action to create a directory, which will throw, and user needs to handle that. It should not close the container. I will mention this in here that user probably wants to handle that error with like an empty catch as they the sub directory already exists and then can just get it by calling get()